node/modules/storageminer.go

```diff
@@ -147,7 +148,7 @@ func StorageNetworkName(ctx helpers.MetricsCtx, a v1api.FullNode) (dtypes.Networ
 	return a.StateNetworkName(ctx)
 }
 
-func SealProofType(maddr dtypes.MinerAddress, fnapi v1api.FullNode) (abi.RegisteredSealProof, error) {
+func SealProofType(maddr dtypes.MinerAddress, fnapi daemonapi.Daemon) (abi.RegisteredSealProof, error) {
```

Comment:
Let's copy over this function to Curio deps to fix the failing tests. I think that's the cheapest solution here instead is a complicated FX dependency fix.